src/components/Disturber.tsx

```diff
@@ -102,13 +112,17 @@ export const Disturber = ({ navigation, publicJsonFile }: Props) => {
               </TouchableOpacity>
             )}
 
-            {!!picture && (
-              <Image
-                source={picture}
-                borderRadius={normalize(8)}
-                aspectRatio={picture.aspectRatio || { HEIGHT: 0.7, WIDTH: 1 }}
-                resizeMode="cover"
-              />
+            {!!pictures?.length && (
+              <View style={styles.carouselContainer}>
+                <ImagesCarousel
+                  aspectRatio={aspectRatio || { HEIGHT: 0.7, WIDTH: 1 }}
+                  autoplayInterval={autoplayInterval}
+                  data={pictures}
+                  isDisturber
+                  navigation={navigation}
+                  refreshTimeKey={`publicJsonFile-${publicJsonFile}`}
+                />
+              </View>
```

Comment:
if there is no image we should consider equal spacing on top like on bottom

<img width="373" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/975a93d5-f287-4bee-add0-5af05f9fabc4">

Reply:
In addition if possible I would reconsider using the 'big' button in line 158, since it design wise dosent really fit in when the text of the button is this much bigger than the other text of the disturber

Reply:
just checked again, in relation to the button of the news below it's also a big difference

Reply:
I didn't make any changes for the button because it was supposed to be done like this as a design in the first place. Later, the button can be updated when releasing according to the request.